source/components/simpleCardList/simpleCard.ts

```diff
@@ -81,10 +81,6 @@ export class SimpleCardComponent<T> extends FormComponent implements OnInit {
 		return canClose;
 	}
 
-	saveForm(): boolean {
-		return this.boolean.toBool(this.submit());
```

Comment:
It looks like the boolean service is no longer being used so the import statement and DI can be removed as well.